build/NuSpecs/UmbracoCms.Core.nuspec

```diff
@@ -23,40 +23,23 @@
       <dependency id="AutoMapper" version="[7.0.1,7.999999)" />
       <dependency id="ClientDependency" version="[1.9.6,1.999999)" />
       <dependency id="ClientDependency-Mvc5" version="[1.8.0,1.999999)" />
-      <dependency id="CSharpTest.Net.Collections" version="[14.906.1403.1082,14.999999)" />
-      <dependency id="Examine" version="[1.0.0-beta025,1.999999)" />
       <dependency id="HtmlAgilityPack" version="[1.7.2,1.999999)" />
```

Comment:
Is both HtmlAgilityPack and ImageProcessor required by Core? It might be that both are required but wanted to double check